crates/egui-winit/src/lib.rs

```diff
@@ -1310,11 +1327,17 @@ fn process_viewport_command(
         ViewportCommand::InnerSize(size) => {
             let width_px = pixels_per_point * size.x.max(1.0);
             let height_px = pixels_per_point * size.y.max(1.0);
-            if window
-                .request_inner_size(PhysicalSize::new(width_px, height_px))
-                .is_some()
+            if let Some(_inner_size) =
+                window.request_inner_size(PhysicalSize::new(width_px, height_px))
             {
-                log::debug!("ViewportCommand::InnerSize ignored by winit");
+                // ex) linux
+                log::info!("ViewportCommand::InnerSize, not will be delivered later with the [WindowEvent::Resized]");
+
+                info.inner_rect = math_inner_rect(window, Some(pixels_per_point));
+                info.outer_rect = math_outer_rect(window, Some(pixels_per_point));
+            } else {
+                // ex) Windows, MacOS
+                log::info!("ViewportCommand::InnerSize, will be delivered later with the [WindowEvent::Resized]");
```

Comment:
When `request_inner_size` returns `None`, it worked as it should, so we shouldn't log anything (or log it at `trace`)

Reply:
> When `request_inner_size` returns `None`, it worked as it should, so we shouldn't log anything (or log it at `trace`)

Yes, this part will only be logged when Some().